google/cloud/bigtable/table.h

```diff
@@ -504,7 +504,7 @@ class Table {
    * @param row_key the row to read.
    * @param filter a filter expression, can be used to select a subset of the
    *     column families and columns in the row.
-   * @returns a tuple, the first element is a boolean, with value `false` if the
+   * @returns A tuple, the first element is a boolean, with value `false` if the
```

Comment:
The `@returns` ones I'm a little less clear on.  For example, this generates something that looks like ...

> **Returns**
>     a tuple, the first element is a boolean, with value false if the row does not exist.
>     If the first element is true the second element has the contents of the Row.
>     Note that the contents may be empty if the filter expression removes all column
>     families and columns.

Is the `Returns` a heading, or the first word of a sentence?  If the latter we don't want to capitalize the "a".  But, if the former, we should do more than just capitalize the "a".

So, I guess I would suggest leaving the `@returns` for now.